packages/waggle/src/nodeTest/testConnect.test.ts

```diff
@@ -197,9 +199,9 @@ describe('Nodes connections', () => {
     await sleep(40_000)
   })
 
-  test('5 nodes connect each other - no tor, providing only one bootstrap multiaddress', async () => {
-    const timeout = 360_000
-    const noOfNodes = 2
+  test.skip('5 nodes connect each other - no tor, providing only one bootstrap multiaddress', async () => {
```

Comment:
Why this is skipped?

Reply:
Because it's not a real-app scenario, now we rely on custom orbitdb strucutre for getting info about peers. I think it's what we just decided on meeting that those tests are not that important